protogen/proto/buf.gen.yaml

```diff
@@ -0,0 +1,35 @@
+version: v1
+plugins:
+  - name: go
+    path: ../../.bingo/protoc-gen-go
+    out: ../gen/
+    opt:
+      - paths=source_relative
+
+  - name: micro
+    path: ../../.bingo/protoc-gen-micro
+    out: ../gen/
+    opt:
+      - paths=source_relative
+
+  - name: microweb
+    #path: ../../.bingo/protoc-gen-microweb
+    path: ../../../protoc-gen-microweb/protoc-gen-microweb
```

Comment:
this might work in CI but breaks when generating locally 🤔

Reply:
You're right. I forgot to revert that change. It must pick the version from .bingo, which is already updated with what we need.